v2/src/translations/en_US.json

```diff
@@ -121,6 +122,14 @@
     },
     "block": {
         "lastBlockTitle": "Last Blocks",
+        "size": "Size",
+        "uncle": "Uncles",
```

Comment:
can you redo uncles please. I dont like how its a single going into plural. 
shoudl either be :
`"uncle": "Uncle | Uncles"`
OR
`"uncles": "Uncles.`
1st example is better ( since when you add another Uncle string you dont have to do plural changes to the original and you can keep them together)